Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs (Dev)/Performance Analysis for a Group of VMs (Dev).workbook

```diff
@@ -8,286 +8,240 @@
         "crossComponentResources": [
           "value::selected"
         ],
+        "parameters": [],
+        "style": "above",
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
```

Comment:
it seems that you have a parameters-0 step at the top of this workbook with no content that is always hidden?

Reply:
Thanks, removed.